### PR TITLE
Add zalgo-text extension

### PR DESCRIPTION
**Pull request by petermiles**
## Description

This extension is a simple translator from normal text to Zalgo text.

## Screencast

https://user-images.githubusercontent.com/11052820/222618683-11543e87-f787-467a-8f3f-f7c802391c97.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by petermiles** [comment]
@raycastbot Just wondering if there's an update here :)

File: extensions/zalgo-text/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "icon": "command-icon.png",
+  "author": "petermiles",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "G̙͖̬̳̜͛̓͘e̶̟ͧ͗n̷̳̤̉ͩ̓͗̈́̇ͪ͡ē̵̫͓̳̹̊ṟ̷ͩ͋a̱̟͍͉̼͖ͤͫͥ̒̿́t̪̥̼͍͖͓̳͒ͫ͂̐ͪ̿e̒̒͏̬̰̠̖̝̘̯ͅ",
```
Comment (pernielsentikaer) [inline extensions/zalgo-text/package.json:15]: Transform to Zalgo

File: extensions/zalgo-text/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      "description": "Convert text to Z̬͚̭̤͓̟͖͐͒ȃ̮̙̯͈͍͇͒ͤ̓ͤ́ͨͅͅl̷̐̍̃̊͏̝ğ̡̪̪͓͈͙̘̽͐͒̆́ö̮̳̪̯̟̩̭̎ͮ̇̿̀͢ ̥͎̂̿͋͋́̀̕͢T̴̷̳̟̟͎͛͌̔̈̇̓ę̠̊͑͊̉̽̓̑̒x̸̠̭͎͍̋̏̇ͯ͋̍ͣt̷̶̤͇̠̜̩̯̦͑̈́ͤ̿͛ͤ̀",
```
Comment (pernielsentikaer) [inline extensions/zalgo-text/package.json:16]:
```suggestion
      "description": "Convert text to Zalgo",
```
You could also add an [argument](https://developers.raycast.com/information/lifecycle/arguments)

File: extensions/zalgo-text/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <>
+      <Form
+        actions={
+          <ActionPanel>
+            <Action.SubmitForm onSubmit={handleSubmit} />
+          </ActionPanel>
+        }
+      >
+        <Form.Description text="Provide the text you'd like to convert to Zalgo" />
+        <Form.TextField id="textfield" title="Text field" placeholder="Enter text" defaultValue="Raycast is cool!" />
```
Comment (pernielsentikaer) [inline extensions/zalgo-text/src/index.tsx:26]:
```suggestion
        <Form.TextField id="textfield" title="Text" placeholder="Enter text" defaultValue="Raycast is cool!" />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 😊

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Pull request merged**